io-engine/src/grpc/v1/stats.rs

```diff
@@ -278,6 +323,18 @@ async fn get_stats(
     Ok(IoStats::from(ExternalType((name, uuid, stats))))
 }
 
+/// Returns IoStats for a given Lvol(Replica).
+async fn replica_stats(lvol: Lvol) -> Result<ReplicaIoStats, CoreError> {
+    let stats = lvol.as_bdev().stats_async().await?;
+    let io_stat =
+        IoStats::from(ExternalType((lvol.name(), lvol.uuid(), stats)));
+    let replica_stat = ReplicaIoStats {
+        stats: Some(io_stat),
+        entity_id: Lvol::get_blob_xattr(lvol.blob_checked(), "entity_id"),
```

Comment:
is this reading from the metadata in memory? I hope it's not causing a read IO..

Reply:
Ill check. You mean if this can alter read stats?

Reply:
It doesnt change any ready stats for the replica when lvol xattr is read.